packages/block/src/block.ts

```diff
@@ -322,8 +323,9 @@ export class Block {
    * Validates the uncle's hash.
    */
   validateUnclesHash(): boolean {
-    const raw = rlp.encode(this.uncleHeaders.map((uh) => uh.raw()))
-    return toBuffer(keccak256(raw)).equals(this.header.uncleHash)
+    const uncles = this.uncleHeaders.map((uh) => uh.raw())
```

Comment:
This is a total nit but should we replace 'uh' with `uncleHeader` just to make it clear.  Sometimes these short acronyms aren't obvious.  My first glance at this had me wondering "uh...what is 'uh'!?'"

Reply:
Works for me.